mitosheet/src/components/taskpanes/UpdateImports/UpdateImportCard.tsx

```diff
@@ -128,6 +152,25 @@ const UpdateImportCard = (props: {
         })
     }
 
+    const getSnowflakeDropddownItemOrEmptyElement = (): JSX.Element =>  {
+        if (props.dataframeCreationData.step_type === 'snowflake_import') {
+            return (
+                <DropdownItem
+                    title='Enter Snowflake Credentials'
```

Comment:
Make the dropdown wider or come up with a shorter name so this doesn't get cut off